pontoon/base/views.py

```diff
@@ -1245,3 +1245,21 @@ def request_projects(request, locale):
 def get_csrf(request):
     """Get CSRF token."""
     return HttpResponse(request.csrf_token)
+
+
+@login_required
+def user_settings(request):
+    """View responsible for user settings for e.g. like the order of suggestions in a locale tab."""
+    if request.method == 'POST':
+        form = forms.UserLocalesSettings(request.POST, instance=request.user.profile)
+        if form.is_valid():
+            form.save()
+            messages.success(request, 'Updated user preferences.')
+            return redirect('/')
```

Comment:
It should take us back to the previous page if openned from the translate view or the profile page.